routes/accounts.js

```diff
@@ -1,12 +1,10 @@
 const express = require( 'express' )
+const router = express.Router()
 
-const encryptPassword = require( '../auth/encryptPassword' )
 const passport = require( '../auth/passport' )
-const RegistrationEmail = require( '../src/mail/registration_email.js' )
-
+const { register } = require( './accounts/accounts_route' )
```

Comment:
The name of this file is misleading - it's not the accounts route, it's the register function... I would rename to simply `register.js` under the accounts folder, and ensure that registering is that module's responsibility.